webflow-hacks/form-hacks/17/17.ts

```diff
@@ -0,0 +1,50 @@
+document.addEventListener('DOMContentLoaded', () => {
+  //assign data attributes to constants
+  const FORM_BUTTON_UP_SELECTOR = '[fs-hacks-element="hack17-counter-button-up"]';
+  const FORM_BUTTON_DOWN_SELECTOR = '[fs-hacks-element="hack17-counter-button-down"]';
+  const COUNTER_INPUT_SELECTOR = '[fs-hacks-element="hack17-counter-input"]';
+
+  /*
+In the video, the classes are referencing "Hack 16"
+This Hack launched as "Hack 17"
+Please note the change of classes from 16 to 17
+*/
+
+  // select elements with their respective data attributes
+  const counterInputs = document.querySelectorAll<HTMLSelectElement>(COUNTER_INPUT_SELECTOR);
+  if (!counterInputs) return;
+
+  const incrementButtons = document.querySelectorAll<HTMLSelectElement>(FORM_BUTTON_UP_SELECTOR);
+  if (!incrementButtons) return;
+
+  const decrementButtons = document.querySelectorAll<HTMLSelectElement>(FORM_BUTTON_DOWN_SELECTOR);
+  if (!decrementButtons) return;
+
+  // loop through each counter input
+  counterInputs.forEach((item, index) => {
+    // declare individual components
+    const counterInput = counterInputs[index];
```

Comment:
This `counterInput` variable is the same one that the `item` variable, no need to redeclare.